features/upgrade/logging/upgrade.feature

```diff
@@ -6,88 +6,80 @@ Feature: Logging upgrading related features
-    # check cron jobs
+    Given I have "json" log pod in project "logging-upg-prep-1"
+    And I have "json" log pod in project "logging-upg-prep-share"
+    Given I run the :oadm_groups_new admin command with:
+      | group_name | project-group-share |
+      | user_name  | <%= user(1, switch: false).name %> |
+    Given I switch to cluster admin pseudo user
+    Given I have clusterlogging with persistent storage ES
+    Then I wait for the project "logging-upg-prep-1" logs to appear in the ES pod
     When I check the cronjob status
     Then the step should succeed
 
     Given I switch to the first user
     When I login to kibana logging web console
+    Given I have index pattern "*app"
+    Then I can display the pod logs of the "logging-upg-prep-1" project under the "*app" pattern in kibana
+    Then I close the current browser
+    Given I run the :policy_add_role_to_group client command with:
+      | group_name | project-group-share |
+      | role | edit |
     Then the step should succeed
-    Given the "logging-upgrade-data-check" project is deleted
+    Given I switch to the second user
+    Given I login to kibana logging web console
+    Given I have index pattern "*app"
+    Then I can display the pod logs of the "logging-upg-prep-share" project under the "*app" pattern in kibana
+    Then I close the current browser
 
   # @case_id OCP-22911
   # @author qitang@redhat.com
   @admin
   @upgrade-check
   @users=upuser1,upuser2
-  @4.9
   Scenario: Cluster logging checking during cluster upgrade
     Given I switch to the first user
-    And I create a project with non-leading digit name
-    And evaluation of `project` is stored in the :proj clipboard
-    Given I obtain test data file "logging/loggen/container_json_log_template.json"
-    When I run the :new_app client command with:
-      | file | container_json_log_template.json |
-    Then the step should succeed
-    # check logging status
-    Given I switch to cluster admin pseudo user
-    And I use the "openshift-logging" project
-    Given I wait for the "fluentd" daemonset to appear up to 300 seconds
-    And I wait until ES cluster is ready
-    And I wait until fluentd is ready
-    And I wait until kibana is ready
+    Given I create a project with non-leading digit name
+    Then evaluation of `project.name` is stored in the :proj1 clipboard
+    And I have "json" log pod in project "<%= cb.proj1 %>"
+    Given I wait for clusterlogging to be functional in the project
     # check the logs collected before upgrading
-    Given I wait for the project "logging-upgrade-data-check" logs to appear in the ES pod
-    # check if logging stack could gather logs
-    And I wait for the project "<%= cb.proj.name %>" logs to appear in the ES pod
-    And evaluation of `cb.doc_count` is stored in the :docs_count_1 clipboard
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-logging" project
```

Comment:
@anpingli No need to add these 2 steps, they're executed in `I wait for clusterlogging to be functional in the project`